### PR TITLE
build(model_prices_and_context_window.json): update Anyscale model list

### PR DESCRIPTION
**Pull request by danielbichuetti**
## Title

Add some new Anyscale models

## Type

🆕 New Feature

## Changes

Updated the JSON with new Anyscale models

### CODE REVIEW

File: model_prices_and_context_window.json [hunk 1]
```diff
@@ -3396,6 +3396,15 @@
...
         "supports_function_calling": true
       },
+      "anyscale/mistralai/Mixtral-8x22B-Instruct-v0.1": {
+        "max_tokens": 65536,
+        "max_input_tokens": 65536,
+        "max_output_tokens": 65536,
+        "input_cost_per_token": 0.00000090, 
+        "output_cost_per_token": 0.00000090,
+        "litellm_provider": "anyscale", 
+        "mode": "chat"
```
Comment (ishaan-jaff) [inline model_prices_and_context_window.json]: please add `source` , `supports_function_calling` for all of these

**Review by ishaan-jaff — CHANGES_REQUESTED** [review]
1 change

File: model_prices_and_context_window.json (hunk 1 above)
Comment (danielbichuetti) [reply model_prices_and_context_window.json]: Just set it up for the models which support it, either via tools or json schema.

image

**Comment by danielbichuetti** [comment]
Would you recommend applying the deprecation list for now? Or better to wait a longer time because someone could still be using it?

https://docs.anyscale.com/preview/endpoints/text-generation/deprecations

**Review by ishaan-jaff — APPROVED** [review]

**Comment by ishaan-jaff** [comment]
> Or better to wait a longer time because someone could still be using it?

Let's wait - don't want to cause a breaking change for anyone

**Pull request merged**